.github/workflows/python.yml

```diff
@@ -120,26 +115,29 @@ jobs:
         run: |
           echo "::set-output name=python-version::$(awk 'FNR == 2' .python-version)"
 
-
       # Until GH composite actions can use `uses`, we need to setup python here
       - uses: actions/setup-python@v2
         with:
           python-version: ${{ steps.python-version.outputs.python-version }}
 
-      - name: Setup sentry env
-        uses: ./.github/actions/setup-sentry
-        id: setup
-        with:
-          python: 3
+      - name: Setup pip
+        uses: ./.github/actions/setup-pip
+        id: pip
 
       - name: pip cache
-        uses: actions/cache@v1
+        uses: actions/cache@v2
         with:
-          path: ${{ steps.setup.outputs.pip-cache-dir }}
+          path: ${{ steps.pip.outputs.pip-cache-dir }}
           key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements-*.txt') }}
           restore-keys: |
             ${{ runner.os }}-pip-
```

Comment:
Missing your py3 cache key.

Reply:
Fixed in [`70427e5` (#21295)](https://github.com/getsentry/sentry/pull/21295/commits/70427e5b74189c4c9e1bf4f97760d4ce9f3df297)